packages/driver/src/multi-domain/events/test.ts

```diff
@@ -0,0 +1,9 @@
+export const handleTestEvents = (Cypress: Cypress.Cypress) => {
```

Comment:
I'm curious, why do we need initialize some of these events? Isn't this handled by creating the Cypress instance for the spec bridge?

Reply:
The test runner is a singleton that only exists in any meaningful way in the primary driver. These events are needed so state gets updated properly between tests, since the spec bridge persists between tests.